app/views/ProjectEdit/Users/UserGroupList/AddUserGroupModal/index.tsx

```diff
@@ -31,25 +31,33 @@ import NonFieldError from '#components/NonFieldError';
 import styles from './styles.css';
 
 interface UserGroupMini {
-    id: number;
-    title: string;
+    id: string;
+    usergroup: {
+        id: string;
+        title: string;
+    }
 }
 
 interface UserGroupMembership {
-    id: number;
-    title: string;
-    usergroup: number;
+    id: string;
+    usergroup: {
+        id: string;
+        title: string;
+    }
 }
 
 const usergroupKeySelector = (d: UserGroupMini) => d.id;
-const usergroupLabelSelector = (d: UserGroupMini) => d.title;
+const usergroupLabelSelector = (d: UserGroupMini) => d.usergroup.title;
```

Comment:
we are using `usergroup.title` for label and `id` for id. There is `usergroup.id` field too. Please check which id is the valid one.